### PR TITLE
Doc 3062v

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

**Comment by tonyjhillman** [comment]
@cb-docs-robot rebuild

File: content/architecture/core-data-access-compression.dita [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+   <title>
+    Where Compression is Used
+   </title>
+   
+   <p>
+    <image href="./images/compression-overview-diagram.png" id="compression-overview-diagram" align="left" width="600" />
+   </p>
+   
+   <p>
+    Clients based on the Couchbase SDK (<b>1</b>), nodes within a cluster that participate
```
Comment (jimwwalker) [inline content/architecture/core-data-access-compression.dita]: I find that this image and description jumps in far too deep too early (this was an image intended for the engineering team design). For example I'm not sure of the benefit of describing a number of the internal compression aspects (e.g. 4) here. Also the description ordering maybe clearer if it follows the numerical order (1 stuff 2 other stuff 3 the other stuff). Once the externally facing compression is described, then maybe we can deep-dive into showing all the places we compress (e.g. confusingly with mode=OFF we still compress to disk).

File: content/architecture/core-data-access-compression.dita [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+   
+   <p>
+    The compression of data allows RAM and disk-space to be used with increased efficiency. It may
+    also reduce consumption of network bandwidth. Higher consumption of CPU resources may result.
+   </p>
+   
+   <p>
+    Compression, provided by the open-source library
+    <xref href="http://en.wikipedia.org/wiki/Snappy_(compression)" scope="external" format="html">Snappy</xref>,
+    is applied per bucket, according
```
[the remark this answers is not in the document]
Comment (tonyjhillman) [reply content/architecture/core-data-access-compression.dita]: Okay I'll modify.

File: content/architecture/core-data-access-compression.dita (hunk 1 above)
Comment (tonyjhillman) [reply content/architecture/core-data-access-compression.dita]: Yes, I agree that this need to be developed. I guess I've been just trying to get the basic accuracy established for now, and so I just dropped the existing image in to expedite. Anyway, for immediate purposes, I can either leave the image and description as they are for now - and develop them post-DP, or just cut the section completely. On balance, the image did help me when I first came to this topic, but I can go either way.

File: content/architecture/core-data-access-compression.dita (hunk 2 above)
[the remark this answers is not in the document]
Comment (tonyjhillman) [reply content/architecture/core-data-access-compression.dita]: Compression, provided by the open-source library
    <xref href="http://en.wikipedia.org/wiki/Snappy_(compression)" scope="external" format="html">Snappy</xref>,
    is applied to documents based both on the client's capabilities and on the compression <i>mode</i> established
    by the user for the given bucket.

**Comment by tonyjhillman** [comment]
Closes DOC-3213

**Pull request merged**